kornia/contrib/models/rt_detr/architecture/resnet_d.py

```diff
@@ -0,0 +1,80 @@
+"""ResNet-D as described in https://arxiv.org/abs/1812.01187.
+
+Based on the code from https://github.com/pytorch/vision/blob/v0.15.1/torchvision/models/resnet.py
+https://github.com/PaddlePaddle/PaddleDetection/blob/release/2.6/ppdet/modeling/backbones/resnet.py
+"""
+from __future__ import annotations
+
+from torch import nn
+
+from kornia.core import Module, Tensor
+from kornia.core.check import KORNIA_CHECK
+
+from .common import ConvNormAct
+
+
+class BottleNeckD(Module):
```

Comment:
Let’s consider later adding this one level upper so that we can use for other tasks like simple classification